server/sqlflowserver_test.go

```diff
@@ -119,7 +117,9 @@ func TestSQL(t *testing.T) {
 	stream, err := c.Run(ctx, &pb.Request{Sql: testErrorSQL})
 	a.NoError(err)
 	_, err = stream.Recv()
-	a.Equal(status.Error(codes.Unknown, fmt.Sprintf("run error: %v", testErrorSQL)), err)
+	a.Error(err)
+	// FIXME(typhoonzero): find out why the error changed.
```

Comment:
The cause is the `testErrorSQL` is appended with a `;`. So it won't go into the mockRun `case testErrorSQL`. Therefore no error is returned. 

https://github.com/sql-machine-learning/sqlflow/blob/229937ac9a6ed26101bff9552b269f8855abe2b7/server/sqlflowserver_test.go#L52-L53

Reply:
Done